<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8" />
  <title>Analysis and Math</title>
  <meta content="width=device-width, initial-scale=1.0" name="viewport" />
  <meta content="Free Website Template" name="keywords" />
  <meta content="Free Website Template" name="description" />

  <!-- Favicon -->
  <link href="img/favicon.ico" rel="icon" />

  <!-- Google Font -->
  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700&display=swap"
    rel="stylesheet" />

  <!-- CSS Libraries -->
  <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" rel="stylesheet" />
  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.0/css/all.min.css" rel="stylesheet" />
  <link href="lib/animate/animate.min.css" rel="stylesheet" />
  <link href="lib/owlcarousel/assets/owl.carousel.min.css" rel="stylesheet" />
  <link href="lib/lightbox/css/lightbox.min.css" rel="stylesheet" />

  <!-- Template Stylesheet -->
  <link href="/css/style.css" rel="stylesheet" />
</head>

<body>
  <!-- Nav Bar Start -->
  <nav class="navbar navbar-expand-lg navbar-light position-fixed">
    <div class="container  justify-content-between">
        <a class="nav-item nav-link navbar-brand" href="/index.html#home">KOFFI IDONGESIT</a>
        <button class="navbar-toggler bg-white" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse flex-grow-0" id="navbarSupportedContent">
            <ul class="navbar-nav me-auto">
                <a class="nav-item nav-link" href="/index.html#home">Home</a>
                <a class="nav-item nav-link" href="/index.html#about" >About</a>
                <a href="/index.html#experience" class="nav-item nav-link">WHY ME</a>
                <a href="/index.html#blog" class="nav-item nav-link">Articles</a>
                <a href="/index.html#testimonial" class="nav-item nav-link">Reviews</a>
            </ul>
        </div>
    </div>
</nav>
  <!-- Nav Bar End -->

  <!-- Blog Start -->
  <div class="blog sub-blog" id="blog">
    <div class="container">
      <div class="section-header text-center wow zoomIn" data-wow-delay="0.1s">
        <!-- <p class="para">From Blog</p> -->
        <!-- <br /> -->
        <!-- <a class="btn" href="/sub_Blog/research.html"><i class="fa fa-angle-left"></i>Back</a> -->
      </div>
      <div class="p-3 write-ups wow zoomIn" data-wow-delay="0.1s">
           <h4 class="">
            Using Fuzzy Equivalence Relations to Model Position Specificity in Sequence Kernels
        </h4>
            <h5 class="text-center">Abstract</h5>
            <p>
                One of the most fundamental issues in computational biology is the classification of biological sequences, particularly nucleotide (DNA and RNA) and amino acid (protein) sequences. Sequence statistics were widely used in the early days of computational biology. Hidden Markov Models (HMM) and Artificial Neural Networks (ANN) were used to improve these methods subsequently (ANN). SVM classifiers are essentially linear classifiers in their most basic form. SVMs differ from other linear classifiers such as perceptrons or logistic regressors in that they maximize the margin between the classes to define the classification function – a principle that is known to be optimal in terms of generalization error limitations. This paper replicates previously published sequence kernels based on the occurrence of specified patterns as well as some position-specific variants. We use fuzzy equivalence relations to model position specificity to develop a generalization of position-specific sequence kernels. This is not just a fascinating link, but it also leads to the creation of new kernels, such as the Elin kernel. We discovered that these kernels enable an explicit representation, allowing for improved computing efficiency and feature extraction.
            </p>
            
            <h5 class="text-center">
                Introduction
            </h5>
            <p>
                One of the most fundamental issues in computational biology is the classification of biological sequences, particularly nucleotide (DNA and RNA) and amino acid (protein) sequences. Sequence statistics were widely used in the early days of computational biology. Hidden Markov Models (HMM) and Artificial Neural Networks (ANN) were used to improve these methods subsequently (ANN). Support Vector Machines (SVM) for sequence classification have grown in popularity over the last decade (Scholkopf et al., 2004). SVMs have outperformed all rival methods in numerous tasks, including computational biology, but also many other disciplines. SVMs are now widely regarded as the most powerful class of classification methods, and this claim is justified (Durbin et al., 1998). SVMs have been effectively applied to a wide range of sequence classification tasks, from promoter and splice site detection (both on DNA data) to protein fold and secondary structure prediction (both on amino acid sequences; Ratsch and Sonnenburg, 2004; Ratsch and Sonnenburg, 2005; Meinicke et al., 2004). 
            </p>
            <p>
                SVM classifiers are essentially linear classifiers in their most basic form. SVMs differ from other linear classifiers such as perceptrons or logistic regressors in that they maximize the margin between the classes to define the classification function – a principle that is known to be optimal in terms of generalization error limitations (Cortes and Vapnik, 1986; Vapnik, 1998). Because linear SVMs only compute scalar products with input vectors, they enable the so-called kernel trick, which involves replacing scalar products with a kernel, which is a nonlinear two-place function that is positive semi-definite. Support vector machines can be used for nearly any type of data, including raw sequence data, signals, pictures, or graphs, thanks to the usage of kernels. All that is required is a suitable kernel. It's no surprise, then, that one of the most important research areas in sequence classification with SVMs is the development of adequate sequence kernels. The majority of these sequence kernels are based on comparing sub-sequences and do not take into account the places of these occurrences. However, in many cases, the presence of a given sub-sequence is only significant if it occurs at a specific location or region.
            </p>
            
        



      </div>
      <div class="align-content-center mt-2 text-center wow zoomIn" data-wow-delay="0.1s">
        <a class="btn" href="/sub_Blog/research.html"><i class="fa fa-angle-left"></i>Back</a>
      </div>
    </div>
  </div>
  <!-- Blog End -->

  <!-- Footer Start -->
  <div class="footer wow fadeIn" data-wow-delay="0.3s">
    <div class="container-fluid">
        <div class="container contact-bottom">
           <div class="contact-info">
            <div class="contact-us-h1">
                <h6 class="text-white">Contact Us</h6>
            </div>
            <p class="text-white mt-4">
               <b>Email:</b> koffijnr@gmail.com
            </p>
           </div>

           <div class="bottom-writer">
            <h6 class="text-white mb-4">Writer</h6>
            <b class="text-white">
                Koffi Idongesit
            </b>
           </div>

           <div class="hq-location">
            <h6 class="text-white mb-4">HQ LOCATION</h6>
            <p class="text-white">
                Patricia C. Amedee 4401 Waldeck New Street  <br>
                Grapevine Nashville, TX 76051
            </p>
           </div>
            <!-- <div class="footer-info">
                <h2>Koffi Idongesit</h2>
                <div class="footer-menu">
                    <h6>HQ Location</h6>
                    <p class="contact-us">
                        Patricia C. Amedee 4401 Waldeck New Street <br> Grapevine Nashville, TX 76051
                    </p>
                    <p>+234-706-455-1848</p>
                    <p>koffijnr@gmail.com</p>
                </div>                       
            </div> -->
        </div>
        <div class="container copyright">
            <p>&copy; <a href="#">Your Site Name</a>, All Right Reserved | Designed By <a href="https://ezekielelom.com">Ellovick</a></p>
            <button class="btn back-to-top"><i class="fa fa-chevron-up"></i></button>                    
        </div>
    </div>
</div>
  <!-- Footer End -->

  <!-- Back to top button -->
  <a href="#" class="btn back-to-top"><i class="fa fa-chevron-up"></i></a>

  <!-- Pre Loader -->
  <!-- <div id="loader" class="show">
         <div class="loader"></div>
     </div> -->

  <!-- JavaScript Libraries -->
  <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.bundle.min.js"></script>

  <!-- <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous"> -->

  <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.10.2/dist/umd/popper.min.js"
    integrity="sha384-7+zCNj/IqJ95wo16oMtfsKbZ9ccEh31eOz1HGyDuCQ6wgnyJNSYdrPa03rtR1zdB"
    crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.min.js"
    integrity="sha384-QJHtvGhmr9XOIpI6YVutG+2QOK9T+ZnN4kzFN1RtK3zEFEIsxhlmWl5/YESvpZ13"
    crossorigin="anonymous"></script>
  <script src="lib/easing/easing.min.js"></script>
  <script src="lib/wow/wow.min.js"></script>
  <script src="lib/waypoints/waypoints.min.js"></script>
  <script src="lib/typed/typed.min.js"></script>
  <script src="lib/owlcarousel/owl.carousel.min.js"></script>
  <script src="lib/isotope/isotope.pkgd.min.js"></script>
  <script src="lib/lightbox/js/lightbox.min.js"></script>

  <!-- Contact Javascript File -->
  <script src="mail/jqBootstrapValidation.min.js"></script>
  <script src="mail/contact.js"></script>

  <!-- Template Javascript -->
  <script src="js/main.js"></script>
</body>

</html>